README.md

```diff
@@ -9,19 +9,60 @@ This offers a OCR-D compliant workspace processor for some of the functionality
 This processor only operates on the text line level and so needs a line segmentation (and by extension a binarized 
 image) as its input.
 
-## Example Usage
+## Installation
+
+### From PyPI
+
+:construction: :construction: :construction: :construction: :construction: :construction: :construction:
+
+```
+pip install ocrd_calamari
+```
+
+### From Repo
 
 ```sh
-ocrd-calamari-recognize -p test-parameters.json -m mets.xml -I OCR-D-SEG-LINE -O OCR-D-OCR-CALAMARI
+pip install .
 ```
 
-With `test-parameters.json`:
+To install the calamari with the GPU version of Tensorflow:
+
+```sh
+pip install 'calamari-ocr[tf_cpu]'
```

Comment:
* I have *never* installed calamri-ocr[tf_cpu], just `tensorflow-gpu==1.14.0 calamari-ocr==0.3.5`
* It says "GPU version" and installs `tf_cpu`. Did you mean CPU version? In this case: It is not necessary, tensorflow-gpu 1.14.0 falls back to CPU. (For TensorfFlow 2.0 the package will be `tensorflow` and it will also fall back to CPU)
* `install the calamari`?
* `requirements.txt` already has this dependency, seperate instructions are unneccessary

Reply:
> I have never installed calamri-ocr[tf_cpu],

It makes even less sense since this is supposed to install the GPU version not the CPU version. Probably best to delete the whole paragraph.